windows_integration/src/auth/wrapper.rs

```diff
@@ -0,0 +1,154 @@
+use once_cell::sync::Lazy;
+use std::{ffi::c_void, time::SystemTime};
+use windows::{
+    core::PSTR,
+    Win32::{
+        Foundation::*,
+        Security::{Authentication::Identity::*, Credentials::STATUS_LOGON_FAILURE},
+        System::Kernel::*,
+    },
+};
+
+use super::package::{AuthError, AuthPackage, AuthInfo};
+
+static mut AUTH_PACKAGE: Lazy<AuthPackage> = Lazy::new(|| AuthPackage::new());
+
+#[tokio::main]
```

Comment:
Forgot to say, you should probably make this "current thlread" to prevent spawning extra threads inside the auth package. Alternately, if you do rt multi-thread, limit it to one created thread.

Reply:
Fixed with [e769657](https://github.com/kanidm/kanidm/pull/1498/commits/e7696572064a414b3b1bfd81f5958edecde9cc9a)